server/rest/dataset.py

```diff
@@ -526,16 +547,62 @@ def _getPostgresGeojsonData(self, item):
             for record in records:
                 key = ''.join([record[x['value']] for x in valueLinks])
                 if key in mappedGeometries:
-                    assembled.append({
-                        'type': 'Feature',
-                        'geometry': mappedGeometries[key],
-                        'properties': record
-                    })
-
+                    assembled.append(
+                        Geojson.Feature(geometry=mappedGeometries[key], properties=record)
+                    )
             if len(assembled) == 0:
                 raise GirderException('Dataset is empty')
 
+            return Geojson.FeatureCollection(assembled)
+
+    @access.public
+    @autoDescribeRoute(
+        Description('Calculate bounding box of a dataset.')
+        .modelParam('id', model='item', level=AccessType.READ)
+        .errorResponse('ID was invalid.')
+        .errorResponse('Read access was denied on the parent folder.', 403))
+    def getBound(self, item, params):
```

Comment:
You may want to add server tests for `getBound` and `download`.  They get tested via the client tests, but aren't reported as covered.